pkg/controller/disruption/disruption.go

```diff
@@ -34,6 +34,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/apimachinery/pkg/util/wait"
+	corev1apply "k8s.io/client-go/applyconfigurations/core/v1"
```

Comment:
The PR currently does not revert the change for Taint manager, but I think it should. The code is now moved to https://github.com/kubernetes/kubernetes/blob/c9158e9a19ef8aaf6c53f6293d1f86371fe8e8a6/pkg/controller/tainteviction/taint_eviction.go#L139

Reply:
Oh, I see we have the code in taint_eviction. Not sure how I missed it or it was added later. Feel free to resolve.

Reply:
Thanks for the review @mimowo , tbh I almost missed this one 😓